### ISSUE TITLE
SVGs are missing role=image

### ISSUE THREAD

**Issue by schalkneethling** [mdn/mdn-fiori#102]
The SVGs used by the new frontend is missing a `role` of image. This stops screen readers from reading out all of the contents of an SVG when embedded in HTML.
